w06-ruby-on-rails/d4-dawn-intro-rails/README.md

```diff
@@ -0,0 +1,195 @@
+
+### Bundler
+
+Bundler is a separate gem from Rails, and can be used outside of
+Rails, but Rails is going to depend on it to manage the RubyGems that the application needs. The first thing that you need to know is that there are two files that matter to bundler: `Gemfile` and `Gemfile.lock`. Gemfile contains configuration information about what
+gems are part of our project, it is similar to the `package.json` file in Node.
+
+Bundler looks at the `Gemfile` loading all the included gems in addition to each's dependencies. It then generates a manifest file that is stored in Gemfile.lock. **Never** edit Gemfile.lock!
+
+So how do you tell Bundler to take your `Gemfile` and turn it into `Gemfile.lock`? Run: `bundle install`. When we create a Rails application it will run this command for us unless we specify otherwise.
+
+## Running Rails (5m)
+
+- Create a new Rails app with `rails new railsFun --skip-activerecord`
+    - the last flag tells the application to not use `activerecord`
+    - `activerecord` is our ORM that manages our Models and our database, we'll learn more about it this afternoon
+- `cd` into your `railsFun` folder and run
+- run `rails server` or just `rails s` and see what happens
+- This will start a server on `localhost:3000` head there and see what it says...
+
+## Rails File Structure (10m)
+
+![Rails File Structure](http://i.imgur.com/whOL4DQ.png)
+
+## Routing (10m)
+
+In `routes.rb` we write logic to map our paths to controllers we will make. See the image below for how the controller actions match up with their respective url paths and HTTP verbs (which combine to define a route).
+
+![Rails Routing](./restful-routing-rails.png)
+
+Let's say when a user sends a `GET` request to the root route, `/`, we want the `WelcomeController`'s `index` method to run. In order to do that we could write:
+
+
+`route.rb`
+
+```ruby
+get "/" => "welcome#index"`
+```
+Try saving the file and heading to your root route. What error do you get?
+
+**Pro tip:** run `rake routes` in your rails app to see a list of all the application's routes. The output will be similar to the image above.
+
+## Controllers (10m)
+
+If you did the last step correctly, you should see an error message: `*uninitialized constant WelcomeController*`. This means that we need to create a controller with the name `welcome` as that is where we told our route to go in the first place!
+
+- Run: `rails generate controller welcome`
+
+This will generate a file in the Controllers directory called `welcome_controller.rb`, along with several other directories relating to it, including views, javascripts, stylesheets.
+
+Reload the page again and find a different error message: `*The action 'index' could not be found for WelcomeController*`. We have created the welcome controller correctly, but there is no `index` method defined. Let's make one:
```

Comment:
I don't think the * is rendering here the way you wanted.  Similar comment for line 93